TLM/CSUtil.Commons/Log.cs

```diff
@@ -56,8 +56,21 @@ private enum LogLevel {
 
         static Log() {
             try {
-                if (File.Exists(LogFilename)) {
-                    File.Delete(LogFilename);
+                string dir = Application.dataPath;
+                LogFilePath = Path.Combine(dir, LOG_FILE_NAME);
+                if (File.Exists(LogFilePath)) {
+                    File.Delete(LogFilePath); // delete old file to avoid confusion.
+                }
+
+                var args = Environment.GetCommandLineArgs().ToList();
```

Comment:
No need to convert ToList() here since you don't use it as a list further down in the method.
Better to keep it a string array and use Array.FindIndex(args, a => a == "-logFile") in the next line.

Reply:
this is not part of this review. I updated the branch